### PR TITLE
don't check plugin artifacts built by IntelliJ into git

### LINKED ISSUE #154: Add plugin artifacts to .gitignore
This is for the binary zip files created in IntelliJ, which are currently showing up in the core-plugin and google-login-plugin directories and are not yet excluded.

### PR DESCRIPTION
**Pull request by elharo**
@loosebazooka close #154

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -58,13 +58,14 @@ On Windows you should be able to build the plugins inside IntelliJ as described
 
 To work in IDEA, just "Open" the cloud-tools-for-intellij directory 
 (the root directory cloned from Github) from the IDEA opening screen.
+To build the plugins, select "Prepare All Plugin Modules For Deployment"
```
Comment (patflynn) [inline CONTRIBUTING.md]: should we not be recommending the gradle build for building the zip?

Comment (elharo) [inline CONTRIBUTING.md]: Until we get it working on Windows, Windows users can do this.

Comment (patflynn) [inline CONTRIBUTING.md]: How about we confine this instruction to the build section where the Windows specific mention is? It's not really specific to project importing anyway.

Comment (elharo) [inline CONTRIBUTING.md]: done

On Mon, Nov 2, 2015 at 1:47 PM, Patrick Flynn <notifications@github.com>
wrote:

> In CONTRIBUTING.md
> <https://github.com/GoogleCloudPlatform/gcloud-intellij/pull/171#discussion_r43664478>
> :
>
> > @@ -58,13 +58,14 @@ On Windows you should be able to build the plugins inside IntelliJ as described
> >
> >  To work in IDEA, just "Open" the cloud-tools-for-intellij directory
> >  (the root directory cloned from Github) from the IDEA opening screen.
> > +To build the plugins, select "Prepare All Plugin Modules For Deployment"
>
> How about we confine this instruction to the build section where the
> Windows specific mention is? It's not really specific to project importing
> anyway.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/GoogleCloudPlatform/gcloud-intellij/pull/171/files#r43664478>
> .
>



-- 
Elliotte Rusty Harold
elharo@macfaq.com

**Comment by patflynn** [comment]
LGTM

**Pull request merged**